README.md

```diff
@@ -16,9 +16,11 @@ Install the TensorFlow Datasets and Abseil Python packages with PIP:
 pip install tensorflow-datasets absl-py
 ```
 
+**Note:** the TensorFlow Datasets package is not required when using the C4 dataset in compressed json format.
```

Comment:
I think it could be clearer that there are two alternative ways to obtain the C4 sentences - either via TensorFlow Datasets or via allenai. Could you try to restructure the README to better reflect this? For example, you could rename "Extract C4_200M target sentences from C4" to "Option 1: Extract C4_200M target sentences from TensorFlow Datasets", move the link on how to setup TensorFlow Datasets into that section, and have a new "Option 2: Extract C4_200M target sentences from allenai" section with the new instructions